test/jdk/java/foreign/NativeTestHelper.java

```diff
@@ -61,7 +61,7 @@
 
 public class NativeTestHelper {
 
-    private static final boolean isAixOS = System.getProperty("os.name").startsWith("AIX");
+    public static final boolean isAixOS = System.getProperty("os.name").startsWith("AIX");
```

Comment:
If you're going to make the field public, I think it should follow the convention established by `IS_WINDOWS`, that is, the name should be `IS_AIX`.

Reply:
Remove the change here as we don't have to publicize the field as we go with `MemoryLayout.paddingLayout(C_DOUBLE.byteAlignment() - 1)` in the struct `S9`.